components/inputs/test/input-date.visual-diff.js

```diff
@@ -10,9 +10,13 @@ describe('d2l-input-date', () => {
 
 	before(async() => {
 		browser = await puppeteer.launch();
-		page = await visualDiff.createPage(browser, { viewport: { width: 800, height: 900 } });
```

Comment:
Some of these tests could be a bit too reliant on state caused by a previous test rather than working in isolation, so there was work done here to improve that and to pull out some shared code (e.g., opening with key and click functions). This PR can probably be ignored until I deal with how the visual-diff PR turns out since there should be some changes, mostly naming, and some new goldens.